label_studio/ml_models/serializers.py

```diff
@@ -16,19 +17,103 @@ class CreatedByFromContext:
     def __call__(self, serializer_field):
         return serializer_field.context.get('created_by')
 
-
 class ModelInterfaceSerializer(serializers.ModelSerializer):
     created_by = UserSimpleSerializer(default=CreatedByFromContext(), help_text='User who created Dataset')
+ 
+    class Meta:
+        model = ModelInterface
+        fields = '__all__'
+        read_only_fields = ['created_by', 'created_at', 'updated_at']
+
+    def check_output_classes(self, project, provided_output_classes):
+        labels, _ = get_all_labels(project.label_config)
+        project_output_classes =  sorted(list(set([label for label_list in labels.values() for label in label_list])))
+        print(project_output_classes)
```

Comment:
nit: let's remove these print statements

Reply:
yes sorry about that I thought I had deleted all of them